Sources/Commander/AsyncCommand.swift

```diff
@@ -1,5 +1,6 @@
 #if compiler(>=5.5)
 /// A simple AsyncCommandType using a closure
+@available(macOS 12, *)
```

Comment:
Do we need to include the iOS and other platform base versions too? Or does the compiler and runtime support this on older versions of all platforms?

Reply:
I don't have any iOS or WatchOS projects that I can test this with, so I cannot say for sure.

These attributes were needed to get a working build on MacOS Monterey so I would assume we'd need to add them for the other Apple platforms as well.